src/pages/settings.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import PageTemplate from "@/components/templates/PageTemplate";
+import SettingsRead from "@/components/organisms/SettingsRead";
+import SettingsChange from "@/components/organisms/SettingsChange";
+import SettingsDropDown from "@/components/organisms/SettingsDropDown";
+import api from "../lib/axios_settings";
+
+export default function SettingsIndex() {
+  const [settingsView, setSettingsView] =
+    useState<string>("drop down settings");
```

Comment:
Can you manage this status by using number?
ex. 
```
const ReadState = {
  All: 0,
  Unread: 1,
  Read: 2,
} as const;

type ReadState = typeof ReadState[keyof typeof ReadState];
```
[Notifications](https://github.com/team-ondo/frontend/blob/develop/src/components/organisms/Notifications.tsx)

Reply:
Done!